src/components/sale/SalesLeadTable.tsx

```diff
@@ -960,4 +1014,22 @@ const TRANSFER_LEADS = gql`
   }
 `
 
+const GET_MEMBER_APPID = gql`
+  query GET_MEMBER_APPID($managerId: String!) {
+    member(where: { id: { _eq: $managerId } }) {
+      app_id
+    }
+  }
+`
+
+const GET_MANAGER_APPID = gql`
```

Comment:
CamelCase

`GetManagerAppID`

the result is list
could be GetManagerAppIDCollection or GetManagerAppIDList